contrib/debian/control

```diff
@@ -53,14 +57,96 @@ Architecture: any
 Depends: cups,
          ${shlibs:Depends},
          netdata (>= ${source:Version})
-Description: The Common Unix Printing System plugin for metrics collection from cupsd
+Description: The CUPS metrics collection plugin for the Netdata Agent
+ This plugin allows the Netdata Agent to collect metrics from the Common
+ UNIX Printing System.
 
 Package: netdata-plugin-freeipmi
 Architecture: any
 Depends: freeipmi,
          ${shlibs:Depends},
          netdata (= ${source:Version})
-Description: FreeIPMI - The Intelligent Platform Management System.
- The IPMI specification defines a set of interfaces for platform management.
- It is implemented by a number vendors for system management. The features of IPMI that most users will be interested in
- are sensor monitoring, system event monitoring, power control, and serial-over-LAN (SOL).
+Description: The FreeIPMI metrics collection plugin for the Netdata Agent
+ This plugin allows the Netdata Agent to collect metrics from hardware
+ using FreeIPMI.
+
+Package: netdata-plugin-nfacct
+Architecture: any
+Depends: ${shlibs:Depends},
+         netdata (>= ${source:Version})
+Description: The NFACCT metrics collection plugin for the Netdata Agent
+ This plugin allows the Netdata Agent to collect metrics from the firewall
+ using NFACCT objects.
+
+Package: netdata-plugin-chartsd
+Architecture: all
+Depends: bash,
+         netdata (>= ${source:Version})
+Suggests: apcupsd, nut, iw, sudo
+Description: The charts.d metrics collection plugin for the Netdata Agent
+ This plugin adds a selection of additional collectors written in shell
+ script to the Netdata Agent. It includes collectors for NUT, APCUPSD,
+ LibreSWAN, OpenSIPS, and Wireless access point statistics.
+
+Package: netdata-plugin-ebpf
+Architecture: i386 amd64
+Depends: ${shlibs:Depends},
+         netdata-plugin-apps (>= ${source:Version}),
+         netdata-plugin-ebpf-code (>= ${source:Version}),
+         netdata (>= ${source:Version})
+Description: The eBPF metrics collection plugin for the Netdata Agent
+ This plugin allows the Netdata Agent to use eBPF code to collect more
+ detailed kernel-level metrics for the system.
+
+Package: netdata-plugin-ebpf-code
+Architecture: i386 amd64
+Description: Compiled eBPF code for the Netdata eBPF plugin
+ This package provides the pre-compiled eBPF code for use by the Netdata
+ eBPF plugin.
```

Comment:
When we compile Netdata on kernels newer than `5.5`, it is possible for users to have eBPF codes without these binaries, because we prefer to use `CO-RE` when `BTF` was compiled with kernel. Probably we need to add an additional information explaining this.

Reply:
Because of how the package itself is being generated, it should be bundling both the CO-RE code _and_ the legacy code. It’s collecting everything under the ebpf.d directory inside the plugins directory.

Reply:
Thanks, I will test it. :handshake: